primitive-types/impls/codec/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.5.1] - 2021-07-02
+### Breaking
```

Comment:
Is it still breaking though?

Reply:
Good catch, fixed! I used `Dependencies` instead, I hope that's okay.